compliance/collection/main.go

```diff
@@ -109,32 +109,16 @@ func startAuditLogCollection(ctx context.Context, client sensor.ComplianceServic
 	return auditReader
 }
 
-// manageSendingToSensor sends everything from sensorC channel to sensor
-func manageSendingToSensor(ctx context.Context, cli sensor.ComplianceServiceClient, sensorC <-chan *sensor.MsgFromCompliance) {
-	for {
-		select {
-		case <-ctx.Done():
-			return
-		case sc := <-sensorC:
-			client, _, err := initializeStream(ctx, cli)
-			if err != nil && ctx.Err() == nil {
-				// error even after retries
-				log.Fatalf("unable to establish send stream to sensor: %v", err)
-			}
-			if err := client.Send(sc); err != nil {
-				log.Errorf("failed sending nodeScanV2 to sensor: %v", err)
-			}
-		}
-	}
-}
-
-func manageReceiveStream(ctx context.Context, cli sensor.ComplianceServiceClient, sig *concurrency.Signal) {
+func manageStream(ctx context.Context, cli sensor.ComplianceServiceClient, sig *concurrency.Signal, sensorC <-chan *sensor.MsgFromCompliance) {
 	for {
 		select {
 		case <-ctx.Done():
 			sig.Signal()
+			log.Infof("Parent context cancelled. Stopping manageStream")
```

Comment:
I think we may remove this log line now. It was useful for understanding the behavior in the debug session, but let's not bother our users with it.